.github/workflows/publish.yaml

```
@@ -22,18 +24,18 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.17.6'
+          go-version: ${{ env.GO_VERSION }}
       - name: Determine version
         run: echo "PIPECD_VERSION=$(git describe --tags --always --abbrev=7)" >> $GITHUB_ENV
 
       # Building backend.
```

Sorry for the many times, I'll leave some nits.
it's up to you, whether choosing `go` or `Go`. lol
```suggestion
      # Building go.
```